extractors/python_functions/metric_extraction/code_snippet.md

```diff
@@ -1,11 +1,10 @@
 ```python
 from quantulum3 import parser
-from typing import List, Tuple
 
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "metric"
 
-def metric_detection(record) -> List[Tuple[str, int, int]]:
+def metric_detection(record):
```

Comment:
detectrion / extraction

Reply:
Changed this to extraction. Because all the modules for the endpoint have the suffix "_extraction" I would say we keep and apply the same naming schema to the source code as well!